config/sync/migrate_plus.migration.import_health_service.yml

```
@@ -0,0 +1,29 @@
+id: import_health_service
+label: Import VAMC SYSTEM HEALTH SERVICE.
+source:
+  plugin: csv
+  path: modules⁩/custom⁩/va_gov_migrate/⁨data/wikes_import.csv⁩
+  delimiter: ';'
```

Your new file is coma separated, so this should be
```suggestion
  delimiter: ','
```